### DISCUSSION TITLE
min burst setting

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#1564] [Ideas] [answered]
This would go next to `min wpm` and `min acc` in the settings:
If you type a single word under this speed, the test fails. 
Even if someone is typing 60 wpm, if each word is typed above the burst minimum (ex: 300 wpm), the test keeps going.
This should only consider raw wpm.

When set, this could have an option called `count slow words as missed words`, which wouldn't fail the test if you typed the word below the `min burst` but would allow you to `practice missed words` for what you're slow *and* inaccurate at. 

I think this feature would be amazing for developing muscle memory for 'word-by-word' typing and for practicing good habits for higher speeds.

**Comment by Miodec** [comment, marked as answer]
Created an issue for it
